CRM/Reltoken/Upgrader.php

```diff
@@ -0,0 +1,172 @@
+<?php
+use CRM_Reltoken_ExtensionUtil as E;
+
+/**
+ * Collection of upgrade steps.
+ */
+class CRM_Reltoken_Upgrader extends CRM_Reltoken_Upgrader_Base {
+
+  private function addCustomData() {
+    $optionValues = civicrm_api3('OptionValue', 'get', [
+      'option_group_id' => 'cg_extend_objects',
+      'name' => 'civicrm_relationship_type'
+    ]);
+    if (!$optionValues['count']) {
+      civicrm_api3('OptionValue', 'create', [
+        'option_group_id' => 'cg_extend_objects',
+        'name' => 'civicrm_relationship_type',
+        'label' => ts('Relationship Type'),
+        'value' => 'RelationshipType',
+      ]);
+    }
+    $customGroups = civicrm_api3('CustomGroup', 'get', [
+      'extends' => 'RelationshipType',
+      'name' => 'reltoken',
+    ]);
+    if (!$customGroups['count']) {
+      $customGroups = civicrm_api3('CustomGroup', 'create', [
+        'extends' => 'RelationshipType',
+        'name' => 'reltoken',
+        'title' => E::ts('Relationship Token Settings'),
+      ]);
+    }
+    $customFields = civicrm_api3('CustomField', 'get', [
+      'custom_group_id' => $customGroups['id'],
+    ]);
+    if (!$customFields['count']) {
+      $customField = civicrm_api3('CustomField', 'create', [
+        'custom_group_id' => $customGroups['id'],
+        'name' => 'display_reltokens',
+        'label' => E::ts('Generate tokens for this relationship'),
+        'data_type' => 'Boolean',
+        'default_value' => 0,
```

Comment:
@MegaphoneJon I'm thinking '1' here would be better for backwards compatibility. So we'd start with supporting all of them, and admins would need to disable specific ones. Thoughts?